R/zchunk_L132.water.demand.manufacturing.R

```diff
@@ -37,40 +37,55 @@ module_water_L132.water.demand.manufacturing_DISABLED <- function(command, ...)
     manufacturing_water_ratios <- get_data(all_data, "water/manufacturing_water_ratios")
 
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+
+    # Pull out global ratio constants for manufacturing water
+    selfTotalRatio <- manufacturing_water_ratios$`self-to-total-ratio`
+    consWithRatio <- manufacturing_water_ratios$`cons-to-with-ratio`
+
+    # Get total industrial energy use for manufacturing water continent regions for 1995
+    L1322.in_EJ_R_indenergy_F_Yh %>%
+      bind_rows(L1322.in_EJ_R_indfeed_F_Yh) %>%
+      select(GCAM_region_ID, X1995) %>%
```

Comment:
We'd like to make it easy for ourselves later, when the data isn't coming in through `temp-data-inject` and presumably will be in long form, with no X's in front of years. So please: 1) in line 37 above, do `get_data(...) %>% gather(year, value, ...) %>% mutate(year = as.integer(substr(year, 2, 5)))` and 2) here and below change code appropriate. I.e., so when we match in the real data later, we only have to delete two lines at top of file.